compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/Psi2IrTranslator.kt

```diff
@@ -72,9 +77,13 @@ class Psi2IrTranslator(
         ktFiles: Collection<KtFile>,
         irProviders: List<IrProvider>,
         linkerExtensions: Collection<IrDeserializer.IrLinkerExtension>,
-        expectDescriptorToSymbol: MutableMap<DeclarationDescriptor, IrSymbol>? = null
+        expectDescriptorToSymbol: MutableMap<DeclarationDescriptor, IrSymbol>? = null,
+        fragmentInfo: EvaluatorFragmentInfo? = null
     ): IrModuleFragment {
-        val moduleGenerator = ModuleGenerator(context, expectDescriptorToSymbol)
+        val moduleGenerator = fragmentInfo?.let {
+            FragmentModuleGenerator(context, fragmentInfo)
```

Comment:
(extremely minor, but since you're using `let`...)
```suggestion
            FragmentModuleGenerator(context, it)
```

Reply:
Done.